examples/remix/README.md

```diff
@@ -0,0 +1,186 @@
+# Use with Remix
+
+This example demonstrates how to use `jest-preview` with [`Remix`](https://remix.run/), bootstrapped using [`create-remix`](https://remix.run/docs/en/v1#getting-started) (without using any built-in templates/stacks).
+
+🚧 This example only demonstrates the simplest use of CSS in a Remix app (global stylesheet). Remix supports [many different ways of styling](https://remix.run/docs/en/v1/guides/styling). Example of more complex ways to style is coming.
+
+## Install Jest
+
+You have to manually install Jest in your Remix app.
+
+First, install `jest` itself. If you are using `jest` version 28 or later, you must install `jest-environment-jsdom` too ([reference](https://jestjs.io/docs/upgrading-to-jest28#jsdom)).
+
+```bash
+npm install --save-dev jest jest-environment-jsdom
+```
+
+Install Babel
+
+```bash
+npm install --save-dev @babel/preset-env @babel/preset-react babel-jest
+```
+
+Config Babel inside `babel.config.js`
+
+```bash
+module.exports = {
+  presets: [
+    ['@babel/preset-env', { targets: { node: 'current' } }],
+    ['@babel/preset-react', { runtime: 'automatic' }],
+  ],
+};
+```
+
+Next, install your favorite testing libraries. This example uses `react-testing-library`, at version 12 specifically ([version 13 wouldn't work](https://stackoverflow.com/questions/71713405/cannot-find-module-react-dom-client-from-node-modules-testing-library-react)). Optionally, you can install its companion [`jest-dom`](https://testing-library.com/docs/ecosystem-jest-dom/) package to add some useful matchers to Jest.
+
+```bash
+npm install --save-dev @testing-library/react@12 @testing-library/jest-dom
+```
+
+## Install & configure Jest Preview
+
+Install Jest Preview
+
+```bash
+npm install --save-dev jest-preview
+```
+
+Enable Jest Preview inside `jest.config.js`. The config options are largely up to you, but remember to remove/add the highlighted options as followed so Jest Preview could work properly.
+
+```diff
+module.exports = {
+  collectCoverageFrom: [
+    '**/*.{js,jsx,ts,tsx}',
+    '!**/*.d.ts',
+    '!**/node_modules/**',
+  ],
+  moduleNameMapper: {
+-   // Handle CSS imports (with CSS modules)
+-   // https://jestjs.io/docs/webpack#mocking-css-modules
+-   '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
+-
+-   // Handle CSS imports (without CSS modules)
+-   '^.+\\.(css|sass|scss)$': '<rootDir>/__mocks__/styleMock.js',
+-
+-   // Handle image imports
+-   // https://jestjs.io/docs/webpack#handling-static-assets
+-   '^.+\\.(png|jpg|jpeg|gif|webp|avif|ico|bmp|svg)$/i': `<rootDir>/__mocks__/fileMock.js`,
```

Comment:
I think users usually want to copy-paste the very looong configuration, so if we change the markdown from `diff` to `js`, it would be very helpful. One more thing is code block on www.jest-preview.com and GitHub have the copy button, so it would help users a lot
![copy button on code block](https://user-images.githubusercontent.com/8603085/169633479-94d917cc-827e-46e2-b1d7-30b2333584d5.png)

One suggestion is to put your message in the comments.